### PR TITLE
Added info for new fellow

### PR DESCRIPTION
**Pull request by GabeSoto**
Adding new iris hep fellow Gabriel Soto

### CODE REVIEW

**Comment by rct225** [comment]
@GabeSoto @klieret Hi Gabriel, you need to a photo to the pull request.  See:  https://iris-hep.org/docs/fellow

File: pages/fellows/2023/GabrielSoto.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+layout: fellow
+pagetype: fellow
+shortname: GabrielSoto
+permalink: N/A
```
Comment (klieret) [inline pages/fellows/2023/GabrielSoto.md:4]: Permalink is `/fellows/GabrielSoto.html`

File: pages/fellows/2023/GabrielSoto.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+fellow-name: Gabriel J. Soto
+title: Gabriel J. Soto - IRIS-HEP Fellow
+active: True
+dates:
+  start: 2023-04-24
+  end: 2023-06-09
+  photo: /assets/images/team/Gabriel-Sot0.png
+institution: University of California, Davis
+e-mail: gjsoto@ucdavis.edu
+focus-area: N/A
```
Comment (klieret) [inline pages/fellows/2023/GabrielSoto.md:14]: ssc

File: pages/fellows/2023/GabrielSoto.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+project_title: Integrating a Lesson on REANA into the HSF Analysis Preservation Training
+project_goal: >
+The project aim of this project is to test REANA and its training material. This study will result in improving the workshop, widening its usability, and mitigating difficult readability.
+mentors:
+  - Kilian Lieret (Princeton University)
+  - Tibor Simko (CERN)
+proposal: /assets/pdf/fellows-2023/proposal-gabriel-soto.pdf
+presentation: N/A
```
Comment (klieret) [inline pages/fellows/2023/GabrielSoto.md:22]: `[]`

File: pages/fellows/2023/GabrielSoto.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+current_status: TBA
```
Comment (klieret) [inline pages/fellows/2023/GabrielSoto.md:23]: leave blank

**Review by klieret — CHANGES_REQUESTED** [review]

**Comment by klieret** [comment]
> @GabeSoto @klieret Hi Gabriel, you need to a photo to the pull request. See: [iris-hep.org/docs/fellow](https://iris-hep.org/docs/fellow)

Hi @rct225 : The photo was already added in #1777

**Pull request merged**